src/services/user/services/MailRegistrationLink.js

```diff
@@ -40,48 +40,48 @@ class SendRegistrationLinkService {
 		let totalMailsSend = 0;
 
 		try {
-			const { userIds } = data;
+			const { userIds = [], users = [] } = data;
```

Comment:
i would accept only userIds or users and work with users during execution. when instead userids are given, load users and continue with them early. do not accept userIds and users to be present. the error handling might be removed.